### PR TITLE
Navbar fixes.

### LINKED ISSUE #147: Navbar takes Username and Currency
**Describe the bug**
Navbar takes hard-coded values instead of the current logged-in user information.

**To Reproduce**
1. Login.
2. View hard-coded values at the top right in the navbar.

**Expected behavior**
Should import logged in user's name and currency.

**Screenshots**
Screenshot (39)

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by blaiadam**
**Description**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

NEEDS TO UPDATE WITHOUT CHANGING WEBPAGES.

Resolves #147 

**Type of change**
Check options that are relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Fix or feature that would cause existing functionality to not work as expected
- [ ] This change requires a update in the design document

**Steps to Verify Functionality**
1. Start website and login.
2. Check username and money in top left.
3. Update user money on backend and check change by switching pages.

**Final Checklist:**
Go down the list and check them off.

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the design document (if applicable)
- [x] My changes generate no new warnings
- [x] Pulled updated master branch (if changed since branch creation) and corrected any conflicts, if any occur.

### CODE REVIEW

File: src/Main.css [hunk 1]
```diff
@@ -152,11 +152,20 @@ body {
...
+.popup input {
+	margin-top: 0px;
+}
+
 .popupbtn {
     color: #04213F;
     background: #04CCFF;
     border: 2px solid #04CCFF;
-    border-radius: 8px;
+	border-radius: 8px;
```
Comment (SecondThread) [inline src/Main.css:167]: fix indentation

File: src/globalComponents/Navbar.js [hunk 2]
```diff
@@ -39,12 +39,47 @@ class Navbar extends React.Component<Props, State> {
 	constructor(props: Props) {
 		super(props);
 
+		// Get initial money from cookies.
+		const cookies = new Cookies();
 		this.state = {
-			userName: "FRICK", // NEED API CALL FOR CURRENT USER
-			userCurrency: 145555 // NEED API CALL FOR USER CURRENCY
+			userName: cookies.get('userName'),
```
Comment (SecondThread) [inline src/globalComponents/Navbar.js:45]: Did you test that this works when there are no cookies set?

**Review by SecondThread — APPROVED** [review]
Please address comments before pushing. Mostly looks good though

File: src/globalComponents/Navbar.js (hunk 2 above)
Comment (blaiadam) [reply src/globalComponents/Navbar.js:45]: I'll try that now.

Comment (blaiadam) [reply src/globalComponents/Navbar.js:45]: image

Here's the error I get when no cookie is set.

**Pull request merged**